README.md

```diff
@@ -2,7 +2,7 @@
 
 [![Build Status][travisci-badge]][travisci-builds]
 
-The source for the Picnic Hiring Experience document.
+The source for the Picnic Hiring Experience document. You can open the Live document here at [hiring.picnic.tech][live-url].
```

Comment:
```suggestion
The source for the Picnic Hiring Experience document. View the live document at [hiring.picnic.tech][live-url].
```

(I wonder whether we can find a more descriptive label than `live-url` :thinking:.)

Reply:
Updated with new wording, hope it descriptive enough.